core/src/mindustry/world/meta/StatValues.java

```diff
@@ -390,7 +390,7 @@ public static <T extends UnlockableContent> StatValue ammo(ObjectMap<T, BulletTy
                     }
 
                     if(type.status != StatusEffects.none){
-                        sep(bt, (type.status.minfo.mod == null ? type.status.emoji() : "") + "[stat]" + type.status.localizedName);
+                        sep(bt, (type.status.minfo.mod == null ? type.status.emoji() : "") + "[stat]" + type.status.localizedName + "[] ~ [stat]" + (type.statusDuration / 60f) + "[] " + Core.bundle.get("unit.seconds"));
```

Comment:
Needs cast to int. Displaying raw floats is a bad idea, and nobody cares if something is 5.5 seconds rather than 5 seconds.